raiden/api/rest.py

```diff
@@ -971,7 +975,7 @@ def get_raiden_events_payment_history_with_timestamps(
     def get_raiden_internal_events_with_timestamps(
         self, limit: Optional[int], offset: Optional[int]
     ) -> Response:
-        assert self.raiden_api.raiden.wal
+        assert self.raiden_api.raiden.wal, "Raiden Service has to be initialized"
```

Comment:
I wonder if we even should have this check here

Reply:
I'm not convinced either, but I want to err on the cautious side for this PR, since it's quite large. Debugging problems introduced in large PRs is unpleasant.